tests/shared/reducers/message.test.js

```diff
@@ -34,4 +37,34 @@ describe("reducers/app", () => {
       message: null,
     });
   });
+
+  describe("test addErrorToState function", () => {
+    it("addErrorMessage with a valid string message", () => {
```

Comment:
you should write sentences starting with `it` (which is already added with the `it` function).

```
it("accepts an error as a string", () => {
```